lib/charms/opensearch/v0/constants_charm.py

```diff
@@ -72,6 +72,7 @@
 BackupRelDataIncomplete = "Backup relation data missing or incomplete."
 BackupRelUneligible = "Only orchestrator clusters should relate to backup relation."
 SecretAccessError = "Failed to access secret, please check permissions."
+SmtpRelationInvalid = "SMTP relation must be created with the main cluster-orchestrator."
```

Comment:
nit for consistency with the rest of statuses:
```suggestion
SmtpRelationInvalid = "SMTP relation must be created with the main-orchestrator cluster."
```

Reply:
Changed! But this was copied from the [JWTRelationInvalid](https://github.com/canonical/opensearch-operator/blob/e5d4db542d05a38b976eb1269580cee793e1b1f6/lib/charms/opensearch/v0/constants_charm.py#L79) status so should that change too?